### PR TITLE
{Release} Release beta 2.17.10

### PR DESCRIPTION
**Pull request by jiasli**
**Description**

Release beta 2.17.10.

After #16380 is resolved, the version will be like 3.0.0b2170.

### CODE REVIEW

**Review by evelyn-ys — APPROVED** [review]

**Review by fengzhou-msft — APPROVED** [review]

File: src/azure-cli-core/setup.py [hunk 1]
```diff
@@ -54,7 +53,6 @@
     'msrest>=0.4.4',
     'msrestazure>=0.6.3',
     'paramiko>=2.0.8,<3.0.0',
-    'PyJWT',
```
Comment (jiasli) [inline src/azure-cli-core/setup.py:57]: Fix:

> msal 1.5.1 requires PyJWT[crypto]<2,>=1.0.0, but you'll have pyjwt 2.0.0 which is incompatible.

File: src/azure-cli/setup.py [hunk 2]
```diff
@@ -131,7 +131,6 @@
     'azure-synapse-accesscontrol~=0.2.0',
     'azure-synapse-artifacts~=0.3.0',
     'azure-synapse-spark~=0.2.0',
-    'cryptography>=2.3.1,<3.0.0',
```
Comment (jiasli) [inline src/azure-cli/setup.py:134]: Fix:

> pyopenssl 20.0.1 requires cryptography>=3.2, but you'll have cryptography 2.9.2 which is incompatible.

Comment (jiasli) [reply src/azure-cli/setup.py:134]: There is already an ongoing PR #15687 to bump `cryptography` to `3.2`.

**Comment by yonzhan** [comment]
Release

**Pull request merged**